.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "flexbox/build.gradle" }}
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "flexbox/build.gradle" }}
+      - run:
+          name: Run Tests
+          command: ./gradlew lint test
```

Comment:
Shouldn't this be `./gradlew lint connectedAndroidTest`?

Reply:
The tests are executed in the Firebase Test Lab so I want to build just apks here.